lib/features/cleanup/config.dart

```diff
@@ -253,13 +250,12 @@ class CleanupConfigState extends ConsumerState<CleanupConfig> {
 
     return Column(
       children: [
+        // Space to the left of the configs.
```

Comment:
This is wrong? It is space to the top. The next space below is to the left? Why the change?